draft-ietf-dnsop-alt-tld.xml

```diff
@@ -160,14 +163,17 @@
       unique among similar groups and, ideally, descriptive.
       Developers are wholly
       responsible for dealing with any collisions that may occur under .alt.</t>
-
-      <t>This document creates an IANA registry for specification documents that
-      use the .alt pseudo-TLD. The intention of the registry is to let developers
-      of non-DNS protocols using the .alt pseudo-TLD know which other developers
-      are using names under .alt. It is possible for multiple different protocols
-      to use the same names as each other. Because there is no requirement or expectation
-      that developers of non-DNS protocols will use the registry, there is no
-      priority given to names that appear in the directory. </t>
+      
+      <t>(This paragraph is here to meet the requirements of <xref target="RFC6761"/>.)
+      Application software that uses alternative namespaces in .alt are expected
+      to have their own processing rules for their own names, probably in specialized
+      resolver APIs, libraries, and/or application software.
+      Users might or might not recognize that names in the .alt pseudo-TLD are special.
+      Caching DNS servers and authoritative DNS servers will treat all names in the
+      .alt pseudo-TLD just as they would any other name whose TLD does not appear in the
+      global DNS root.
+      DNS server operators and DNS registries/registrars for the global DNS will never
+      names in the .alt pseudo-TLD because .alt will not be in the global DNS root.</t>
```

Comment:
Erm. Some comments:
1: "will never names in" - I'm unclear what word is missing, but I'm probably fine with it (perhaps "register")

2:  "will not be in the global DNS root." -> "will not exist in the global DNS root"? 

3: "(This paragraph is here to meet the requirements of <xref target="RFC6761"/>.)" Perhaps "(This paragraph exists to meet the requirements of <xref target="RFC6761"/>, Section 5.)"

Note that the only one I actually care about is 1 (the others are just readability suggestions)

Reply:
Fixed by 80a4ddc